src/ast/query.rs

```diff
@@ -654,14 +654,16 @@ impl fmt::Display for Values {
 pub struct SelectInto {
     pub temporary: bool,
     pub unlogged: bool,
+    pub table: bool,
```

Comment:
I like that this follows the rest of the crate's design to mirror the input syntax in the crated AST

👍